journal.html

```diff
@@ -0,0 +1,200 @@
+                            fill="black" />
+                        <path
+                            d="M16.3 3.50303C-5.43333 14.5697 -5.43333 45.2364 16.3 56.1697C26.0333 61.103 33.7667 61.103 43.2333 56.1697C65.2333 44.703 64.8333 13.503 42.5667 3.10303C33.3667 -1.16364 25.5 -1.0303 16.3 3.50303ZM41.5 6.70303C59.7667 15.6364 59.7667 44.1697 41.5 53.103C17.9 64.4364 -5.83333 38.4364 7.9 16.303C15.1 4.70303 29.1 0.703032 41.5 6.70303Z"
+                            fill="black" />
+                        </svg>
+                        <div class="label" id="happy-label">Happy</div>
+                    </label>
+                    </div>
+                    <div>
+                    <label>
+                        <input type="radio" name="feeling" value="joyful" >
+                        <svg class="face-icon" width="60" height="60" viewBox="0 0 60 60" fill="none"
+                        xmlns="http://www.w3.org/2000/svg">
+                        <path
+                            d="M16.6667 24.1333C15.2 25.7333 15.3333 26.5333 17.0667 28.4C19.2 30.4 19.4667 30.4 21.4667 28.4C24 26 23.0667 22 20.2667 22C19.2 22 17.6 22.9333 16.6667 24.1333Z"
+                            fill="black" />
+                        <path
+                            d="M37.6 22.8C36 24.4 36.5333 28.4 38.4 29.0667C41.2 30.1333 43.6 28 43.0666 25.2C42.6666 22.6667 39.2 21.2 37.6 22.8Z"
+                            fill="black" />
+                        <path
+                            d="M16 36.9333C17.6 42.5333 24.1334 47.3333 30 47.3333C35.8667 47.3333 42.4 42.5333 44 36.9333C44.8 34 44.8 34 30 34C15.2 34 15.2 34 16 36.9333ZM38.2667 40.9333C36.2667 43.0667 33.7334 44 30 44C26.2667 44 23.7334 43.0667 21.7334 40.9333L18.8 38H30H41.2L38.2667 40.9333Z"
+                            fill="black" />
+                        <path
+                            d="M16.3 3.50303C-5.43333 14.5697 -5.43333 45.2364 16.3 56.1697C26.0333 61.103 33.7667 61.103 43.2333 56.1697C65.2333 44.703 64.8333 13.503 42.5667 3.10303C33.3667 -1.16364 25.5 -1.0303 16.3 3.50303ZM41.5 6.70303C59.7667 15.6364 59.7667 44.1697 41.5 53.103C17.9 64.4364 -5.83333 38.4364 7.9 16.303C15.1 4.70303 29.1 0.703032 41.5 6.70303Z"
+                            fill="black" />
+                        </svg>
+            
+                        <div class="label" id="joyful-label">Joyful</div>
+                    </label>
+                    </div>
+                </div>
+                <button type="submit">Submit</button>
+            </section>
+        </section>
+
+        <!-- Task Modal -->
+        <!-- Hidden until user adds/edits a task-->
+        <section id="task-modal" class="modal">
+            <div class="modal-content">
+                <form class="task">
+                    <label for="task-desc">Task Description</label>
+                    <textarea id="task-desc" name="task-desc" rows="2" cols="30"></textarea>
+                    <br>
```

Comment:
You can also delete the `<br>` tags under the modal. I know it makes it look a little nicer when previewing right now but each section should be laid out on the page using CSS later on. It'd also be helpful to group the labels and inputs into `<div>`s